src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,41 +1,71 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { currentTodoActions } from '../../features/currentTodo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
 
 export const TodoModal: React.FC = () => {
+  const [choosedTodoUser, setChoosedTodoUser] = useState<User | null>(null);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo) || null;
+
+  useEffect(() => {
+    if (currentTodo) {
+      getUser(currentTodo.userId).then(user => {
+        setChoosedTodoUser(user);
+      });
+    }
+  }, [currentTodo]);
+
+  const handleDeleteCurrentTodo = () => {
+    dispatch(currentTodoActions.delete());
```

Comment:
Verify that the `delete` action in `currentTodoActions` is correctly implemented to remove the current todo from the state.